Assets/MixedRealityToolkit.Examples/Demos/UX/BoundsControl/Scripts/BoundsControlExampleTest.cs

```diff
@@ -0,0 +1,270 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.UI.Experimental;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using System.Text;
+using TMPro;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
```

Comment:
Move to experimental. And while we are at it, I think it would be nice to rename this to "BoundsControlRuntimeExample" and remove the word "test", since it is not a playmode or edit mode test.